is/object_of_test.ts

```diff
@@ -98,4 +98,46 @@ Deno.test("isObjectOf<T>", async (t) => {
       >(true);
     }
   });
+
+  await t.step("if 'predObj' has prototype properties", async (t) => {
```

Comment:
It should explain the behavior like "prototype properties of 'predObj' does not affect the predication"